packages/airnode-deployer/test/mock-utils.ts

```diff
@@ -0,0 +1,24 @@
+// NOTE: This file is referenced as a pathGroup pattern in .eslintrc (import/order)
+
+import fs from 'fs';
+
+// Declare originalFs outside of mockReadFileSync to prevent infinite recursion errors in mockReadFileSync.
+const originalFs = fs.readFileSync;
+
+/**
+ * Mocks the fs library if the file path includes the specified file path substring
+ * and otherwise returns the original content.
+ */
+export const mockReadFileSync = (
+  filePathSubstr: string,
+  mockValue: unknown,
+  options?: { replacer?: (number | string)[] | null; space?: string | number }
+) => {
+  return jest.spyOn(fs, 'readFileSync').mockImplementation((...args) => {
+    const path = args[0].toString();
+    if (path.includes(filePathSubstr)) {
+      return JSON.stringify(mockValue, options?.replacer, options?.space);
```

Comment:
Now that I think of it - we should probably not stringify here as that is the detail that can vary (e.g. you might want to mock this for secrets.env and those should not be stringified).

It would be a bit better if `mockValue` was string and we would call `JSON.stringify` when calling this method.

Reply:
This makes sense, can also get rid of the options then. Will update